### ISSUE TITLE
ControlNET SDXL tutorial - "You are using a model of type clip_text_model to instantiate a model of type"

### ISSUE THREAD

**Issue by alelordelo** [huggingface/diffusers#5997]
### Describe the bug

I am following this tutorial:
https://github.com/huggingface/diffusers/blob/main/examples/controlnet/README_sdxl.md

When running the model with same settings I get this warning:
`You are using a model of type clip_text_model to instantiate a model of type . This is not supported for all configurations of models and can yield errors.`

### Reproduction

Run:
train_controlnet_sdxl.py

With default settings:
export MODEL_DIR="stabilityai/stable-diffusion-xl-base-1.0"
export OUTPUT_DIR="path to save model"

accelerate launch train_controlnet_sdxl.py \
 --pretrained_model_name_or_path=$MODEL_DIR \
 --output_dir=$OUTPUT_DIR \
 --dataset_name=fusing/fill50k \
 --mixed_precision="fp16" \
 --resolution=1024 \
 --learning_rate=1e-5 \
 --max_train_steps=15000 \
 --validation_image "./conditioning_image_1.png" "./conditioning_image_2.png" \
 --validation_prompt "red circle with blue background" "cyan circle with brown floral background" \
 --validation_steps=100 \
 --train_batch_size=1 \
 --gradient_accumulation_steps=4 \
 --report_to="wandb" \
 --seed=42 \
 --push_to_hub
 
 
 

### Logs

```shell
Detected kernel version 5.4.0, which is below the recommended minimum of 5.5.0; this can cause the process to hang. It is recommended to upgrade the kernel to the minimum version or higher.
11/30/2023 10:12:20 - INFO - __main__ - Distributed environment: NO
Num processes: 1
Process index: 0
Local process index: 0
Device: cuda

Mixed precision type: fp16

You are using a model of type clip_text_model to instantiate a model of type . This is not supported for all configurations of models and can yield errors.
You are using a model of type clip_text_model to instantiate a model of type . This is not supported for all configurations of models and can yield errors.
{'thresholding', 'dynamic_thresholding_ratio', 'clip_sample_range', 'variance_type'} was not found in config. Values will be initialized to default values.
{'reverse_transformer_layers_per_block', 'dropout', 'attention_type'} was not found in config. Values will be initialized to default values.
11/30/2023 10:12:28 - INFO - __main__ - Initializing controlnet weights from unet
```

### System Info

python3.10
diffusers==0.24.0.dev0

**Comment by sayakpaul** [comment]
That's a harmless warning.

**Comment by alelordelo** [comment]
Thanks!😊 Sent from my iPhoneOn 30 Nov 2023, at 11:40, Sayak Paul ***@***.***> wrote:﻿
That's a harmless warning.
